### PR TITLE
Fix some tests

### PR DESCRIPTION
**Pull request by szabolcs-instructure**
Fix some tests. 

refs: MBL-16214
affects: Teacher, Student
release note: none
test plan:

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Teacher Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


<details><summary>:x: Swift lint</summary>


    ❌ /Users/vagrant/git/Student/StudentUITests/Syllabus/SyllabusTests.swift:33:97: Comma Spacing Violation: There should be no space before and one after any comma. (comma)
    
    
</details>


[MBL-16214](https://instructure.atlassian.net/browse/MBL-16214)

#### Affected Apps: Teacher, Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

File: Student/Student.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3387,6 +3387,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 0.0.1;
+				OTHER_SWIFT_FLAGS = "$(inherited) -D COCOAPODS";
```
Comment (vargaat) [inline Student/Student.xcodeproj/project.pbxproj:3390]: Do we need these flags?

**Comment by davidvarga93** [comment]
Screenshot 2022-09-01 at 10 52 44
There's still 1 failing test, unrelated to this PR, will be fixed separately

File: Student/Student.xcodeproj/project.pbxproj (hunk 1 above)
Comment (szabolcs-instructure) [reply Student/Student.xcodeproj/project.pbxproj]: I reverted these changes.

**Review by vargaat — APPROVED** [review]

**Review by idevainstr — APPROVED** [review]

**Pull request merged**